### DISCUSSION TITLE
Keys for bind

### DISCUSSION THREAD

**Discussion by oRauu** [EveOPlus/eve-o-preview#20] [General]
It is possible (if i dont have programming keyboard) to change F13-F14 default keys for more useful? Like tilda ( ` ) or something like that? I try to use it and EVE-O didnt start (error key in configurate), but with Alt+1 for example it worked well.
Probably some windows key code for tilda (or another keys) to make then useful?

**Comment by EveOPlus** [comment]
This has been covered many times but just noticed it here as well. It's possible if you know what to enter but more importantly is now it can all be done in the UI so you don't need to know what to manually type into the config file.
